pkg/cmd/server/origin/auth_config.go

```diff
@@ -83,11 +83,9 @@ func BuildAuthConfig(masterConfig *MasterConfig) (*AuthConfig, error) {
 	}
 
 	// Build the list of valid redirect_uri prefixes for a login using the openshift-web-console client to redirect to
-	// TODO: allow configuring this
-	// TODO: remove hard-coding of development UI server
 	assetPublicURLs := []string{}
 	if !options.DisabledFeatures.Has(configapi.FeatureWebConsole) {
-		assetPublicURLs = []string{options.OAuthConfig.AssetPublicURL, "http://localhost:9000", "https://localhost:9000"}
+		assetPublicURLs = []string{options.OAuthConfig.AssetPublicURL}
```

Comment:
is that going to delete the existing redirectURI for the embedded console,
do we need to specify both?

On Mon, Sep 12, 2016 at 9:39 AM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/server/origin/auth_config.go
> <https://github.com/openshift/origin/pull/10819#discussion_r78373970>:
>
> >  	assetPublicURLs := []string{}
> >  	if !options.DisabledFeatures.Has(configapi.FeatureWebConsole) {
> > -		assetPublicURLs = []string{options.OAuthConfig.AssetPublicURL, "http://localhost:9000", "https://localhost:9000"}
> > +		assetPublicURLs = []string{options.OAuthConfig.AssetPublicURL}
>
> what are the steps that would need to be run?
>
> after the master starts, add https://localhost:9000 to the redirectURIs
> list
>
> oc patch oauthclient/openshift-web-console -p '{"redirectURIs":["https://localhost:9000"]}'
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10819/files/a0578bff9477ca300b23c0f364bcdf929e9c69d5#r78373970>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABZk7cNJlUNd4-lBSnasn3Ub0ftXo-ZVks5qpVYqgaJpZM4J17f2>
> .
>